roles/manage_cnp/README.md

```diff
@@ -0,0 +1,98 @@
+# manage_cnp
+
+This Ansible Galaxy Role contains task files that assist in the management of CloudNativePG and EDB Postgres for Kubernetes.
+
+**The ansible playbook must be executed under an account that has full
+privileges.**
+
+## Requirements
+
+The only dependencies required for this ansible galaxy role are:
+
+  1. AWS EKS CLI, Azure CLI, or Google Cloud CLI - Depends directly on the target cloud
+  2. Kubectl
+  3. Ansible
+  4. Python packages: openshift, pyyaml, kubernetes, pyhelm
+     Installed with commands:
+     `pip install openshift pyyaml kubernetes`
+     `pip install pyhelm`
+  5. `kubernetes.core.k8s` - Ansible Module - Required for Kubernetes.
+
+## Role variables
+
+The 'cnp_task' must be defined with a value in order to indicate which task in the role will be executed.
```

Comment:
Change to `cnp_task` but also add the possible options to this description.

Reply:
fixed